nova/utils.py

```diff
@@ -364,6 +357,15 @@ def last_completed_audit_period(unit=None, before=None):
     return (begin, end)
 
 
+# Default symbols to use for passwords. Avoids visually confusing characters.
+# ~6 bits per symbol
+DEFAULT_PASSWORD_SYMBOLS = ('23456789',  # Removed: 0,1
+                            'ABCDEFGHJKLMNPQRSTUVWXYZ',   # Removed: I, O
+                            'abcdefghijkmnopqrstuvwxyz',  # Removed: l
+                            CONF.password_additional_symbols,
+                           )
```

Comment:
Does this still work if the config is parsed after `nova.utils` got imported or will it have the default value of `[]` then for `password_additional_symbols`?

Reply:
Good question, I'll have to check

Reply:
Indeed it doesn't work. I moved all the symbols to a config-option instead.